tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -165,10 +167,17 @@ def create_public_release_files(self):
             container_client = self.blob_service_client.get_container_client(
                 "downloads")
 
-        self.upload_files(container_client,
-                          self.all_files_zip_files)
-        self.upload_files(container_client,
-                          self.data_files)
+        try:
+            self.upload_files(container_client,
+                              self.all_files_zip_files)
+        except ResourceExistsError:
```

Comment:
Are you sure you need to wrap every call to `upload_files` with try - except `ResourceExistsError` which I see in several places here?

I thought that within `upload_files` you are ignoring the `ResourceExistsError` using

```
except ResourceExistsError:
    pass
```

Doesn't this mean it won't be re-raised?

Reply:
I see the change you made but you've moved `except ResourceExistsError:` lower down which means now if the create_container fails because the container already exists it will skip uploading files. The container might exist but be empty in which case we still want to upload files.

Reply:
I've reverted the changes & I have readded the `pass ` in the various except blocks as we don't want to do anything when an exception is raised (i.e. a blob container already exists).